stockApp/stockApp/Presentation/View/AssetView.swift

```diff
@@ -0,0 +1,175 @@
+import UIKit
+
+private struct Constants {
+    static let fontSize: Float = 1.0
+    static let fontSizeTitle: Int = 35
+    static let fontSizeButton: Int = 18
+    static let fontSizeDefault: Int = 35
+    static let numberOfLines: Int = 0
+    static let shadowRadius: Int = 1
+    static let shadowOpacity: Float = 1.0
+    static let shadowOffset: Int = 3
+    static let edgeInsetsZero: Int = 0
+    static let imageEdgeInsetsRight: Int = 20
+    static let titleEdgeInsetsRight: Int = -20
+    static let contentEdgeInsetsRight: Int = 10
+    static let searchHeightMultiplier = 0
+    static let searchHeightConstant = 40
+    static let searchWidth = 0.11
+    static let searchTop = 40
+    static let searchTrailing = -40
+    static let searchCornerRadius: Int = 18
+    static let titleTop: Int = 40
+    static let titleLeading: Int = 30
+    static let titleTrailing: Int = 0
+    static let assetsListTop: Int = 30
+    static let assetsListLeading: Int = 10
+    static let assetsListTrailing: Int = -10
+    static let assetsListBottom: Int = -10
+    static let defaultTop: Int = 300
+    static let defaultLeading: Int = 0
+    static let defaultTrailing: Int = 0
+}
+
+// MARK: - Class
+
+class AssetView: UIView {
+
+    // MARK: - Properties
+
+    let searchUIButton: UIButton = {
```

Comment:
```suggestion
    let searchButton: UIButton = {
```

change others please!